FCS-to-CSV v2.0.R

```diff
@@ -9,8 +9,12 @@
 ##### USER INPUT #####
     
     # Install packages if required
-    if(!require('flowCore')) {install.packages('flowCore')}
-    if(!require('Biobase')) {install.packages('Biobase')}
+    # BiocManager is used to provide compatibility for flowCore for R version 3.6 and above.
+    if (!requireNamespace("BiocManager", quietly = TRUE))
+      install.packages("BiocManager")
+    BiocManager::install("flowCore")
+    BiocManager::install("Biobase")
```

Comment:
As far as I can see, this change might break backwards-compatibility with older versions of R.

@tomashhurst Would you accept a pull request with an if statement that checks for the R version and then either uses the current code (R < 3.6) or the code @ghar1821 suggested for higher R versions?

Chris

Reply:
That certainly would! There might be some issues with the Bioconductor end of things, but we can certainly try that out.